geopandas/io/tests/test_sql.py

```diff
@@ -773,6 +773,22 @@ def test_append_with_different_crs(self, engine_postgis, df_nybb):
         with pytest.raises(ValueError, match="CRS of the target table"):
             write_postgis(df_nybb2, con=engine, name=table, if_exists="append")
 
+    @pytest.mark.parametrize("engine_postgis", POSTGIS_DRIVERS, indirect=True)
+    def test_append_without_crs(self, engine_postgis, df_nybb):
+        # This test was included in #3328 when the default value for no
+        # CRS was changed from an SRID of -1 to 0. This resolves issues
+        # of appending dataframes to postgis that have no CRS as postgis
+        # no CRS value is 0.
+        engine = engine_postgis
+        df_nybb.crs = None
```

Comment:
```suggestion
        df_nybb = df_nybb.set_.crs(None, allow_override=True)
```
This is to silence our deprecation warnings.